### DISCUSSION TITLE
我这个经历，有润的可能性吗？

### DISCUSSION THREAD

**Discussion by chainthehell** [The-Run-Philosophy-Organization/run#390] [General]
看来看去都是CS专业牛人或者coding人士，难道其他IT从业者没有可能了吗？
本人34岁，11年国内普通大学文学类学士毕业。毕业就转行做了IT infrastructure，网络、Linux、SDN、数据库、渗透/安全，掌握程度从高到低。有ITIL基础认证、PMP认证、AWS(amazon web service) Advanced Networking-specialty等国际认证，以及1张国内工信部认证。英语6级且看油管技术类视频无压力无需字幕，口语也挺溜。16年到20年做过大大小小的项目6个，金额达6000w人民币左右，大部分是政府单位的信息化项目。
……
就这，有run的可能性吗？请赐教。

**Comment by the-run-philosophy** [comment]
看你这个情况我觉得润澳大利亚很有可能。可以去文件夹里找找澳洲润的经验。
